.github/workflows/main.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           path: /tmp/apt/*.deb
           # The trailing number serves as a version flag that can be incremented
           # to invalidate the cache after changing the list of packages.
-          key: ${{ runner.os }}-${{ steps.get-date.outputs.date }}-apt-3
+          key: ${{ github.workflow }}-${{ runner.os }}-${{ steps.get-date.outputs.date }}-apt-3
```

Comment:
I guess once your PR that allows to specify lua version is merged we could use the same apt cache for both workflows right ? (didn't cross check the packages thoroughly but I'm guessing the lua5.3 package installed in this workflow is what makes them different)

Reply:
The main workflow installs various packages related to running tests that aren't needed when building docs.

Reply:
Sure, but most of them overlap I guess. Anyway, it's irrelevant for now